src/channel-provider.js

```diff
@@ -0,0 +1,70 @@
+import { ChannelProvider } from '@connext/channel-provider'
+import { setupMultiplex } from './stream-utils'
+
+const SafeEventEmitter = require('safe-event-emitter')
+
+export class TorusChannelRpcConnection extends SafeEventEmitter {
+  constructor(connectionStream) {
+    super()
+    this.promises = {}
+    this.connectionStream = connectionStream
+
+    const self = this
+    // super constructor
+    SafeEventEmitter.call(self)
```

Comment:
This isn't necessary. super() call exists

Reply:
Yeah I thought that was weird but I was just following some pattern I saw somewhere else in the codebase

Reply:
We refactored that to follow es6 now. Please use super() only

Reply:
I already fixed it on https://github.com/torusresearch/torus-embed/pull/65/commits/94c98808c59c74986ddfc486c544931998ea3e3d